### PR TITLE
[TCID-DIR-OP-CSTOR-POOL-MIRROR-RECOMMEND-REBOOT-NODE]Verify mirror cstor pool cluster after node reboot

### PR DESCRIPTION
**Pull request by amitbhatt818**
**What this PR does / why we need it**:

- Verify mirror cstor pool cluster after node reboot

**_Details:_**

**_Steps involved in openebs update in cluster:_**

- **Pre-checks**: 
  - Check whether all the openebs components are in running state or not.
  - DOP should be installed

- **Steps involved in the test case**
  - Invoke API to list recommendations
  - Invoke API to get capacity based recommendations
  - Invoke API to get device based recommendations
  - Invoke API to create mirror cstor pool cluster
- Reboot node and then verify the status of pools

#### Expected output:
- Director should list the supported recommendations without error
- Director should list the capacity based recommendations without error
- Director should list the device based recommendations without error

**Additional Information-** 

| Title | Description |
| --- | --- |
|Assumptions | openebs should be already installed|
||Dop Installed |
| Application Under Test | Verify mirror cstor pool cluster after node reboot |
| Stogare Engine |  |
|Application Used| |
| Openebs Version | 1.9.0 |

**Notes to reviewer**

### CODE REVIEW

**Comment by amitbhatt818** [comment]
> @amitbhatt818 can you please list down the limitations of this testcase.
> What all corner cases are not covered by this testcase?
> e.g. if there are 4 nodes, and cstor pool is on a node that is not restarted due the current random technique of picking a node.

In this test case when we create pools using recommendation it gets created in all the nodes. @AmitKumarDas

File: litmus/director/TCID-DIR-OP-CSTOR-POOL-MIRROR-RECOMMEND-REBOOT-NODE/test.yml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+          
+          - name: Rebooting node
+            shell: sshpass -p {{vm_pass}} ssh {{vm_user}}@{{vm1_ip}} 'reboot'
+
+          - pause:
+              minutes: 3
+
+          when: platform == "RANCHER"
+        
+        - name: fun begins
```
Comment (harshshekhar15) [inline litmus/director/TCID-DIR-OP-CSTOR-POOL-MIRROR-RECOMMEND-REBOOT-NODE/test.yml:171]: @amitbhatt818 please rename this task and name it appropriately.
